sdk/python/tests/integration/online_store/test_universal_online.py

```diff
@@ -510,6 +506,22 @@ async def test_async_online_retrieval_with_event_timestamps(
     await fs.close()
 
 
+@pytest.mark.integration
+@pytest.mark.universal_online_stores(only=["redis", "postgres"])
+async def test_async_online_retrieval_with_event_timestamps(
+    environment, universal_data_sources
+):
+    await _do_async_retrieval_test(environment, universal_data_sources)
+
+
+@pytest.mark.integration
+@pytest.mark.universal_online_stores(only=["dynamodb"])
+async def test_async_online_retrieval_with_event_timestamps_dynamo(
+    environment, universal_data_sources
+):
+    await _do_async_retrieval_test(environment, universal_data_sources)
```

Comment:
@franciscojavierarceo trying to test the theory here ... im not familiar w how the environment reuse works exactly, but if is then separating the tests should work. unless it's cached across tests and not just within a test.